plugins/wiggle/src/DensityRenderer/DensityRenderer.test.js

```diff
@@ -1,56 +0,0 @@
-import SimpleFeature from '@jbrowse/core/util/simpleFeature'
-import DensityRenderer, { configSchema, ReactComponent } from '.'
-
-const pluginManager = {}
```

Comment:
why do we have to delete the density renderer tests?

Reply:
restored along with snpcoveragerenderer test